src/exercise/05.js

```diff
@@ -4,17 +4,21 @@
 import * as React from 'react'
 import '../box-styles.css'
 
-// 💰 Use the className for the size and style (backgroundColor) for the color
-// 💰 each of the elements should also have the "box" className applied
+function Box({size,style,children}) {
+    const className=`box box--${size}`;
+    
+    style.fontStyle="italic";
```

Comment:
I like this much better than the way the extra credit example implemented this part. Much cleaner.

Reply:
I just took a look, and for the purposes of getting the solution, I like mine better. For use in an actual project, theirs is better. Only because it allows the default italic to be overwritten by passing a fontStyle value, since the spread has precedence as it comes after.